src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -651,26 +651,38 @@ public void Cancel()
     {
         _threadingContext.ThrowIfNotOnUIThread();
 
-        // This wait is safe.  We are not passing the async callback to DismissUIAndRollbackEditsAndEndRenameSessionAsync.
-        // So everything in that method will happen synchronously.
-        DismissUIAndRollbackEditsAndEndRenameSessionAsync(
-            RenameLogMessage.UserActionOutcome.Canceled, previewChanges: false).Wait();
```

Comment:
switched off of an explicit Wait (which was not JTF to begin with) to a VerifyCompleted which is more of a contract assertion about something which must be true (and which throws if not).  During 'Cancel', all we're doing is rolling back UI state.  This is all doable synchronously, without need for waits.